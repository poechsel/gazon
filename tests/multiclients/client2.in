login u2
pass p2
whoami
w
exit
