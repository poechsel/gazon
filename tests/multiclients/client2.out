u2
u1 u2
